sklearn/cluster/k_means_.py

```diff
@@ -1262,6 +1264,15 @@ class MiniBatchKMeans(KMeans):
     Notes
     -----
     See http://www.eecs.tufts.edu/~dsculley/papers/fastkmeans.pdf
+
+    See also
```

Comment:
Thank you @nelson-liu and @NelleV for your comments. I decided that the best way would be to follow the official Numpy Documentation standards and placed the `See also` section before `Notes`, since this is the way we want the things to be.

I also made a similar change to `KMeans` docstring in order to harmonize both docs within the same module.